.devcontainer/devcontainer.json

```
@@ -1,17 +1,24 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/cpp
 {
-	"name": "C++",
+	"name": "R-Dev-Env",
 	"build": {
 		"dockerfile": "Dockerfile"
+         },
```

```suggestion
	"image": "ghcr.io/r-devel/r-dev-env:dev-container",
```
It may be faster to pull in the fully configured image rather than building the image from the dockerfile each time.